File: runtimes/py/template.py
import sys
import pickle

{{SOLUTION}}

inputS = """{{INPUT}}"""
expectedS = """{{OUTPUT}}"""
expected  = {{OUTPUT}}

try:
    result  = solution({{INPUT}})
    resultS = repr(result)
    
    #if resultS != expectedS:
    if pickle.dumps(result) != pickle.dumps(expected):
        sys.stderr.write("solution(%s) => %s     Expected: %s\n" % (inputS, resultS, expectedS) )
    else:
        print "solution(%s) => %s     OK!" % (inputS, resultS)
except BaseException as err:
    sys.stderr.write(err)
    sys.stderr.write("\n")

File: runtimes/c/template.c
#include <stdio.h>

{{SOLUTION}}

char* inputS = "{{INPUT}}";
int expected = {{OUTPUT}};
char* expectedS = "{{OUTPUT}}";

int main() {
    int result = solution({{INPUT}});

    if (result != expected) {
        fprintf(stderr, "solution(%s) => %d     Expected: %s\n", inputS, result, expectedS);
    }
    else {
        printf("solution(%s) => %d     OK!\n", inputS, result);
    }
    
    return 0;
}
